airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/test/mock/MockDestinationBackend.kt

```diff
@@ -108,9 +108,7 @@ object MockDestinationBackend {
     }
 
     fun deleteOldRecords(filename: String, minGenerationId: Long) {
-        getFile(filename).removeAll {
-            it.generationId == null || it.generationId!! < minGenerationId
-        }
+        getFile(filename).removeAll { it.generationId == null || it.generationId < minGenerationId }
```

Comment:
actual diff is just removing the `!!`, which was tripping werrorr. no idea how this code _wasn't_ tripping werror previously.